action.yml

```diff
@@ -0,0 +1,55 @@
+inputs:
+  targets:
+    description: 'depviz target (format: \"org1/repo1 org1/repo2 org2 repo1\")' 
+    required: true
+  branch: 
+    description: 'the branch where to push the generated files' 
+    default: 'depviz-generate'
```

Comment:
Add a input for “filters”, maybe a single arg that takes `—filter-foo —filter-bar`